components/mediation/registry/org.wso2.micro.integrator.registry/src/main/java/org/wso2/micro/integrator/registry/MicroIntegratorRegistry.java

```diff
@@ -367,7 +403,7 @@ public RegistryEntry getRegistryEntry(String key) {
             entryEmbedded.setType(MicroIntegratorRegistryConstants.FILE);
             entryEmbedded.setDescription("Resource at : " + url.toString());
             entryEmbedded.setLastModified(urlc.getLastModified());
-            entryEmbedded.setVersion(urlc.getLastModified());
+            entryEmbedded.setVersion(getLastModifiedTimestamp(resolveRegistryURI(key), urlc));
```

Comment:
Would only setting the version is enough? Do not we need to remove the particular key from [synapseConfiguration](https://github.com/wso2/wso2-synapse/blob/e01af546918c60e846fea1898fcc91431b19e90f/modules/core/src/main/java/org/apache/synapse/config/SynapseConfiguration.java#L929)?

Reply:
Yes, it checks for the version https://github.com/wso2/wso2-synapse/blob/master/modules/core/src/main/java/org/apache/synapse/registry/AbstractRegistry.java#L96